src/Microsoft.IdentityModel.JsonWebTokens/Microsoft.IdentityModel.JsonWebTokens.csproj

```diff
@@ -28,12 +27,11 @@
   </ItemGroup>
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'net461' Or '$(TargetFramework)' == 'net462' Or '$(TargetFramework)' == 'net472' Or '$(TargetFramework)' == 'netstandard2.0' Or '$(TargetFramework)' == 'net6.0'">
```

Comment:
`Or '$(TargetFramework)' == 'net6.0'` can likely be removed here, as `System.Text.Json` is inbox since 3.0 (IIRC) and `System.Text.Encodings.Web` probably since the first .NET Core versions 😃

Reply:
@ViktorHofer ^